examples/basic/src/components/ExampleAllFields.js

```diff
@@ -1,14 +1,19 @@
 import React, { Component } from 'react';
-import { Form, Validator, Text, Select, TextArea, Radio, Checkbox, FieldError } from 'react-uforms'
+import { Form, Validator, Text, Select, TextArea, RadioGroup, Radio, Checkbox, FieldError } from 'react-uforms'
 import Code from './Code';
 import Json from './Json';
 
 class ExampleAllFields extends Component {
 
+  static radioItems = [
```

Comment:
I'm not sure `static` is important for this case. I suggest add `const` to render for more readability.